akka-actor/src/main/scala/akka/io/Tcp.scala

```diff
@@ -106,7 +106,7 @@ object Tcp extends ExtensionId[TcpExt] with ExtensionIdProvider {
    * This is the common trait for all commands understood by TCP actors.
    */
   trait Command extends Message with SelectionHandler.HasFailureMessage {
-    def failureMessage = CommandFailed(this)
+    def failureMessage: CommandFailed = CommandFailed(this)
```

Comment:
with scalfix rule?

Reply:
I don't think there will be a scalafix rule for this case.
I don't add explicit type def for all public methods. This case cause dotty compile errors => I manually change.

Reply:
but new code will be added then.

Reply:
> but new code will be added then.

what? sorry I don't understand.

Reply:
How do you keep Akka building with Dotty, if new contributions are only tested with Scalac and not fixed with Scalafix? (Maybe that’s what @hepin1989 is asking, but it’s a good question in either case). Are you adding Dotty to CI? And would Scalafix help port new contributions to Dotty?

Reply:
> Are you adding Dotty to CI?

Yes. I have already described this in the PR description above. Note this image: [![Build Status](https://travis-ci.org/ohze/akka.svg?branch=dotty-wip)](https://travis-ci.org/ohze/akka)

Reply:
ok. I know we should use scalafix and create rules if one don't exist.

But for this very specific case `def failureMessage: CommandFailed = CommandFailed(this)`, I don't think we can create a rule to fix this (maybe I am wrong :D). And we don't have many code like this to create a rule.

Rules are good for other cases though, ex "constructor procedure syntax", or "don't use auto application"